inspirehep/modules/workflows/tasks/refextract.py

```diff
@@ -91,12 +92,22 @@ def extract_journal_info(obj, eng):
         refextract_request_headers = {
             "content-type": "application/json",
         }
-        response = requests.post(
-            "{}/extract_journal_info".format(current_app.config["REFEXTRACT_SERVICE_URL"]),
-            headers=refextract_request_headers,
-            data=json.dumps({"publication_infos": publication_infos, "journal_kb_data": kbs_journal_dict})
-        )
-        extracted_publication_info = response.json()['extracted_publication_infos']
+        try:
+            response = requests.post(
+                "{}/extract_journal_info".format(current_app.config["REFEXTRACT_SERVICE_URL"]),
+                headers=refextract_request_headers,
+                data=json.dumps({"publication_infos": publication_infos, "journal_kb_data": kbs_journal_dict})
```

Comment:
don't think you need `json.dumps` btw

Reply:
maybe we can retry before we fail the workflow? especially if it's timeout or bad gateway?